src/syft/core/common/message.py

```diff
@@ -1,17 +1,19 @@
 from typing import Optional
 
 from syft.core.common.uid import UID
+from syft.core.common.object import ObjectWithID
 from syft.core.io.address import Address
 
 
-class AbstractMessage:
+class AbstractMessage(ObjectWithID):
```

Comment:
We should remove AbstractMessage.

Reply:
Agreed - dont need to block this PR for it - but agreed. i'm down for the string typing stuff.

Reply:
Yeah, sorry unfortunately when I submit my PR's i'm often not awake to make these small changes. So please feel free to make any small adjustments either directly to the PR or on your days updates. Otherwise if the PR is open when I wake up I will make the changes as normal.